src/commands/new.ts

```diff
@@ -144,6 +146,13 @@ export interface Options {
    * @default false
    */
   noTimeout?: boolean
+  /**
+   * Whether or not to remove MobX-State-Tree boilerplate code
+   *
+   * Input Source: `prompt.ask` | `parameter.option`
+   * @default false
+   */
+  removeMST?: boolean
```

Comment:
Despite all other evidence above, we've been trying to go away from boolean options (see the `experimental` flag).

So, maybe consider doing something like:

```
state?: "mst" | "none"
```

Open to other opinions, though. [GAFO](https://capwatkins.com/blog/the-sliding-scale-of-giving-a-fuck) level is 3/10.

Reply:
Actually looking at the rest of the code, perhaps `removeMST` is fine?

Reply:
makes sense, yeah I'll roll with that!

Reply:
Yeah I had a similar thought. `state` sounds like it opens the door for potentially supporting other state options, which we don't want due to maintenance effort. 
And the code itself is similar enough to the `removeDemo` code that I thought it was a decent pattern to copy. 
I might vote for `removeMST` but also low gafo

Reply:
We could also just call the flag `--mst` and default it to true

This way it reads `--mst=false` when you want to opt out via cli (or you hit the question and `--mst=false` will be output on the generated command to run the project generation again).

And then you don't have anything called `state` misleading the user that we'll be open to supporting others out of the box, etc

Reply:
I ended up using `--mst` flag and defaulting to true 👍